test/unit/colorspace_spec.js

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ColorSpace } from '../../src/core/colorspace';
+
+describe('colorspace', function () {
+  it('should be true if decode is not an array', function() {
+    expect(ColorSpace.isDefaultDecode('I am a string', 0)).toBe(true);
```

Comment:
As indicated in https://github.com/mozilla/pdf.js/pull/8611#discussion_r125623641, please use `toBeTruthy()`/`toBeFalsy()` instead.

Also, please don't forget to [squash the commits](https://github.com/mozilla/pdf.js/wiki/Squashing-Commits).